app/_data/tables/install_options.yml

```diff
@@ -12,65 +12,65 @@ features:
         url: /gateway/VERSION/kong-production/install-options/docker/build-custom-images/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: true
       - name: 'Debian'
         url: /gateway/VERSION/kong-production/install-options/linux/debian/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: true
       - name: 'Ubuntu'
         url: /gateway/VERSION/kong-production/install-options/linux/ubuntu/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: true
       - name: 'RedHat'
         url: /gateway/VERSION/kong-production/install-options/linux/rhel/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: true
       - name: 'Amazon Linux 2'
         url: /gateway/VERSION/kong-production/install-options/linux/amazon-linux/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: true
   - name: 'Kubernetes'
     items:
       - name: 'Kubernetes (YAML Manifests)'
         url: /gateway/VERSION/kong-production/install-options/kubernetes/
         oss: true
         enterprise: true
-        support_text: Local Testing
+        support: false
       - name: 'Kubernetes (Helm)'
         url: /gateway/VERSION/kong-production/install-options/helm-quickstart/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: true
       - name: 'OpenShift'
         url: /gateway/VERSION/kong-production/install-options/openshift/
         oss: true
         enterprise: true
-        support_text: Production Support
+        support: false
  
   - name: 'Docker'
     items:
       - name: 'Debian (bullseye-slim)'
         oss: true
-        enterprise: false
-        support_text: Production Support
+        enterprise: true
+        support: true
       - name: 'RHEL (8-ubi)'
         oss: true
-        enterprise: false
-        support_text: Production Support
+        enterprise: true
+        support: true
       - name: 'Alpine'
```

Comment:
Q: I thought we weren't providing official alpine images anymore?

Reply:
We're providing them for testing purposes only. They're not certified for production deployment

Reply:
In that case, shouldn't support be `false`?